### ISSUE TITLE
Implement adding skills page

### ISSUE THREAD

**Issue by JoshSmith** [code-corps/code-corps-ember#7]
We need to implement the design finished in #6.

**Comment by begedin** [comment]
@scarfdestroyer This is the most self-contained task I could find on the ember side, so it would be a good one for you to work on in paralel with the stuff I'm working on.

The design itself isn't ready yet, but the way it would work is clear enough.

1. There is a general list of skills the user can pick from. They work as toggle buttons.
2. User can toggle those skills so they are either selected or not selected. Selecting a skill creates a `UserSkill` record, deselecting it destroys it.

Note that skills can be categorized, so they need to be listed in category groups.

As for the "show me the projects" button, I don't think we can do anything about that yet, since we don't have a project page yet. There will be a task to handle that separately.

![image](https://cloud.githubusercontent.com/assets/1925540/12393679/848d883c-bdf7-11e5-8203-804a6cceb32e.png)

**Comment by begedin** [comment]
@JoshSmith It's not completely clear if this works sort of like LinkedIn, where the user can also type in a skill and
* If the skill already exists, a user skill is created
* If the skill doesn't already exist, both a skill and a user skill are created

My guess for now is "no, we only have predefined skill records the user can pick from", but I wanted to make sure.

@scarfdestroyer Regardless of what Josh's answer is, proceed as if you only have a list of predefined skills and can't create new ones. If there will be actual skill creation, we can do that in a separate task.

**Comment by begedin** [comment]
@JoshSmith I need your input here. I'm not blocked on starting work, but we need to figure out if it will be possible for the user to create new skills this way, or if it's just assigning predefined skills.

By this, I mean, a user want's to specify they're skilled with HTML, but there is no skill with that name in the database. Do we want to allow them to add it, which will create a `Skill` as well as a `UserSkill` record?

**Comment by JoshSmith** [comment]
I think for now just all predefined. We need to come up with a list of them. Maybe use stackshare or something as the basis for adding those initially?

Later we can consider having users add their own.

**Comment by JoshSmith** [comment]
Closed by #163.
